templates/CRM/Mailing/Page/Report.tpl

```diff
@@ -149,7 +145,7 @@
 {/if}
 
 <fieldset>
-<legend>{ts}Content / Components{/ts}</legend>
+<legend>{ts}Content{/ts}</legend>
```

Comment:
I also did a sneaky update here to remove nerdy jargon.

(this does not break translations, the string "Content" already exists)